docs/builds.rst

```diff
@@ -60,6 +63,18 @@ key to ``true`` in order for simexpal to pull the respective files.
 
 If ``recursive-clone`` is not set, it will default to ``false``.
 
+Specifying Local Builds
```

Comment:
Maybe "Local Builds" -> "Builds without Version Control".

Reply:
Done. I also changed this formulation at some other parts of the documentation